### ISSUE TITLE
Laravel 5.6 support

### ISSUE THREAD

**Issue by pedzed** [leemason/tenantable#11]
Any plans for Laravel ~5.4~ ~5.5~ 5.6 support?

**Comment by lloy0076** [comment]
@leemason - actually does this work on **L5.4** at al?

**Comment by johannesschobel** [comment]
Are there plans for L5.6 and the upcoming L5.7 @leemason

**Comment by pedzed** [comment]
This looks like abandonware.

I recommend https://laravel-tenancy.com. They have excellent support on their Discord channel.

It was a bit too bloated for my project, but I have no regrets.

**Comment by johannesschobel** [comment]
@pedzed yeah, i know this project - their support is exceptional good! however, the new `tenancy` package is not "production ready" yet :(

**Comment by pedzed** [comment]
@johannesschobel Just use their older [`Hyn/Multi-Tenant`](https://github.com/hyn/multi-tenant/): https://laravel-tenancy.com/docs/hyn/5.2

It's stable and I am using it with success.

**Comment by johannesschobel** [comment]
i know this package as well @pedzed ..

however, there are a few "quirks", i don't really like with `hyn` - and i am hoping this will change in `tenancy`. For example, i want to define an explicit (!) database-configuration on my `Tenant` models. For example, i want to manually define the database-name, database-host and so on, as this is crucial in my application scenario.

`hyn`, however, auto-generates the database-name based on a `Generator`, automatically creates the database, user and password to access this database. While is this a very powerful approach (i.e., no credentials are stored in the "system" database!), it is not practicable in my situation :cry:

**Comment by pedzed** [comment]
@johannesschobel I think it's more fit to talk about it in the Discord channel.

**Comment by johannesschobel** [comment]
i have already created an issue and got sophisticated response on this one.. But currently i dont feel like to stick to  `hyn` .. maybe i will wait for another 2-3 months with starting with the project.. currently i'm quite busy anyway with finishing my phd, haha :laughing:
